go/utils/config_test.go

```diff
@@ -0,0 +1,134 @@
+package utils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+)
+
+func fakeServer() (*http.Server, int) {
+	http.HandleFunc("/api-token-auth/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Seem that `httptest` is a good choice for testing HTTP servers(server request handlers), but these test cases are test client calls. A "fake" server must be started with partial function to test all the client routines.

But It's a nice package! Will make use of it somewhere else!